### PR TITLE
janet: new recipe

### PR DESCRIPTION
**Pull request by Crestwave**
The patch has already been merged into upstream.

### CODE REVIEW

File: dev-lang/janet/janet-0.5.0.recipe [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+DESCRIPTION="Janet is a functional and imperative programming language \
+and bytecode interpreter. It is a modern lisp, but lists are replaced by \
+other data structures with better utility and performance (arrays, tables, \
+structs, tuples). The language also supports bridging to native code written \
+in C, meta-programming with macros, and bytecode assembly."
+HOMEPAGE="https://janet-lang.org/"
+COPYRIGHT="2019 Calvin Rose and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/janet-lang/janet/archive/v0.5.0.tar.gz"
```
Comment (korli) [inline dev-lang/janet/janet-0.5.0.recipe]: you could use the $portVersion

Comment (Crestwave) [reply dev-lang/janet/janet-0.5.0.recipe]: Whoops! Fixed

File: dev-lang/janet/janet-0.5.0.recipe [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+structs, tuples). The language also supports bridging to native code written \
+in C, meta-programming with macros, and bytecode assembly."
+HOMEPAGE="https://janet-lang.org/"
+COPYRIGHT="2019 Calvin Rose and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/janet-lang/janet/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f5c90122715869657427cbba62809a1d53eaaced2268e7d55202301d65a75076"
+SOURCE_FILENAME="janet-$portVersion.tar.gz"
+PATCHES="janet-0.5.0.patch"
```
Comment (korli) [inline dev-lang/janet/janet-0.5.0.recipe]: $portVersion here also

Comment (Crestwave) [reply dev-lang/janet/janet-0.5.0.recipe]: But it's already upstreamed

Comment (korli) [reply dev-lang/janet/janet-0.5.0.recipe]: you mean there will be a release with these changes?

Comment (Crestwave) [reply dev-lang/janet/janet-0.5.0.recipe]: The next release should have it, I think; it's in master.

File: dev-lang/janet/janet-0.5.0.recipe [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	lib:libjanet$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	janet${secondaryArchSuffix}_devel = $portVersion
+	devel:libjanet$secondaryArchSuffix = $libVersionCompat
+"
```
Comment (korli) [inline dev-lang/janet/janet-0.5.0.recipe]: Missing tab here

Comment (Crestwave) [reply dev-lang/janet/janet-0.5.0.recipe]: Fixed

**Pull request merged**